src/Resources/skeleton/registration/RegistrationController.tpl.php

```diff
@@ -32,10 +35,16 @@ class <?= $class_name; ?> extends <?= $parent_class_name; ?><?= "\n" ?>
 {
 <?php if ($will_verify_email): ?>
     private $emailVerifier;
+<?php if (!$verify_email_with_auth): ?>
+    private <?= $use_typed_properties ? 'EntityManagerInterface ' : null ?>$manager;
+<?php endif; ?>
```

Comment:
I think we should autowire this into the method, not the constructor. That's my default mode - `EmailVerifier` is kind of an exception because we know it will definitely be needed in all methods.

Reply:
done